src/documentation/runners/jstorm.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: default
+title: "JStorm Runner"
+permalink: /documentation/runners/jstorm/
+redirect_from: /learn/runners/jstorm/
+---
+# Using the JStorm Runner
+
+The JStorm Runner can be used to execute Beam pipelines using [JStorm](http://jstorm.io/). 
+
+The JStorm Runner executes Beam pipelines on top of JStorm, providing:
+
+* The at-least-once and exactly-once fault tolerance.
```

Comment:
add "high throughput and low latency"
and more about standalone and integration with Hadoop

Reply:
Done